package.json

```diff
@@ -8,5 +8,6 @@
   "homepage": "https://mainnet.lido.fi",
   "dependencies": {
     "@lido-js/ganache": "=7.9.2-lido"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Please check that this line does not break the scripts repo setup (git clone it from scratch and checkout current branch)